discovery_engine/lib/src/ffi/types/document/document_vec.dart

```diff
@@ -69,8 +69,13 @@ extension DocumentSliceFfi on List<DocumentFfi> {
     return res;
   }
 
-  List<Document> toDocumentList() => asMap()
+  List<DocumentWithActiveData> toDocumentListWithData() => asMap()
```

Comment:
maybe for completeness. or maybe we could use a shorthand like `toActiveDocuments()` or sth similar, if this is not already used in other contexts.
```suggestion
  List<DocumentWithActiveData> toDocumentListWithActiveData() => asMap()
```